docs/network-drivers.md

```diff
@@ -213,6 +213,40 @@ The body of the api call request will look like:
 The brocade driver accepts interface names the same way they would be accepted
 in the console of the switch, ex. ``101/0/10``.
 
+### Dell Networking OS 9 driver
+
+#### switch_register
+
+The ``type`` field for the Dell Networking OS 9 driver has the value:
+
+    http://schema.massopencloud.org/haas/v0/switches/dellnos9
+
+In addition to ``type``, you need to supply ``username``, ``password``,
+``hostname``, and ``interface_type``. ``interface_type`` is of the form
+"GigabitEthernet", "TenGigabitEthernet". Please look at the switch guide for
+more valid interface types.
+
+The API server either runs on port 8008 (HTTP) or 8888 (HTTPS), so be sure to
+specify that in the ``hostname``.
+
+If you have multiple types of ports on the same switch, register the switch
+multiple times with different parameters for ``interface_type``.
```

Comment:
This is kindof gross, is it really much work to do this as port_register?

Reply:
I thought this was reasonable and generally accepted since I got this from the brocade driver.

Doing this in port register means, I would have to read the whole interface_type + interface like `TenGigabitEthernet 1/3`. 

Anyway, making this change would require a significant amount of changes everywhere.

Reply:
Ug, okay, I guess we can live with it. I'm realizing I'm somewhat dissatisfied with the quality of the brocade driver...

Reply:
yeah, I'll avoid copy-pasting in the future.